laurent22/joplin
# Mobile: Resolves #8381: Notify users if there are items that cannot be decrypted

# Summary
On mobile, this pull request shows details of a decryption failure when a user clicks on an encrypted note. For example,
![Screenshot of decryption details screen](https://github.com/laurent22/joplin/assets/46334387/cbf7b6f5-e920-466d-a272-17e31b4d30da)

Clicking "retry" either displays an error message or, if decryption is successful, navigates the user to the content of the decrypted note. 

# To-do
Testing:
1. Break decryption for several notes
2. Click "retry" in the decryption screen for each (an error message should appear)
3. Fix decryption for the notes
4. Try decrypting again using the "retry" button.

- [x] Manually test on Android
- [ ] Manually test on iOS
